src/Css/Rule/Processor.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace TijsVerkoyen\CssToInlineStyles\Css\Rule;
+
+use \TijsVerkoyen\CssToInlineStyles\Css\Property\Processor as PropertyProcessor;
+use TijsVerkoyen\CssToInlineStyles\Css\Specificity\Specificity;
+
+class Processor
+{
+    /**
+     * Split a string into seperate rules
+     *
+     * @param string $rulesString
+     * @return array
+     */
+    public function splitIntoSeparateRules($rulesString)
+    {
+        $rulesString = $this->cleanup($rulesString);
+
+        return (array) explode('}', $rulesString);
+    }
+
+    /**
+     * @param $string
+     * @return mixed|string
+     */
+    protected function cleanup($string)
```

Comment:
I suggest making this private. There is no reason to make a child class replace it